openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -1042,7 +1071,6 @@ data:
                       image: quay.io/openshift-knative/knative-serving-operator:v0.5.1
                       args:
                         - --olm
```

Comment:
The `--olm` option is obsolete

Reply:
Er... actually I just noticed this was for 0.5.1. It may not be obsolete for that version. I can't remember. :)

Reply:
It's not for 0.5.1. This is a straight up generated file.

Reply:
Its presence will break the latest operator.